### DISCUSSION TITLE
classes/class_gpuparticles3d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#193] [User-contributed notes]
# classes/class_gpuparticles3d

Inherits: GeometryInstance3D< VisualInstance3D< Node3D< Node< Object A 3D particle emitter. Description: 3D particle node used to create a variety of particle systems and effects. GPUParticles3D fe...

https://docs.godotengine.org/en/stable/classes/class_gpuparticles3d.html

**Comment by edesdan** [comment]
I think for local_coords is also important to mention that, if set to True,  it will follow parent's scale.

**Comment by Naradoxy** [comment]
https://github.com/godotengine/godot/issues/76360
to anyone having issues setting the color in emit particles, that feature has been broken for years
